docs/docs/query-builder/1-select-query-builder.md

```
@@ -225,6 +225,15 @@ const photosSums = await dataSource
 // result will be like this: [{ id: 1, sum: 25 }, { id: 2, sum: 13 }, ...]
 ```
 
+Note that `getRawMany` also supports the use of `QueryOptions` to return a structured result:
+
+```typescript
+const structured = await dataSource
+	.createQueryBuilder(User, "u")
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)